File: frontend/src/types/job.ts
export interface JobInfo {
  name: string;
  namespace: string;
  state: "running" | "succeeded" | "failed" | "unknown";
  active: number;
  succeeded: number;
  failed: number;
  cron_job: string | null;
  start_time: string | null;
  completion_time: string | null;
  created_at: string | null;
}

File: backend/app/schemas/job.py
from pydantic import BaseModel


class JobInfo(BaseModel):
    name: str
    namespace: str
    state: str
    active: int
    succeeded: int
    failed: int
    cron_job: str | None
    start_time: str | None
    completion_time: str | None
    created_at: str | None
